query.go

```diff
@@ -977,6 +977,48 @@ func (q *Query) BufferVar(attributeName string) ([]uint64, interface{}, error) {
 	return offsets, buffer, nil
 }
 
+// BufferSizeVar returns the size of the backing C buffers for the given variable-length attribute
```

Comment:
Something to consider for later, is in go using raw byte size is unexpected. Go users expect to use element counts. @antalakas and I are planning to eventually cleanup the go api to remove all byte size references and return element counts everywhere we can.

Reply:
@Shelnutt2 updated the PR to return element counts instead of bytes.